new-integration-tests/src/test/resources/apps/clusterview/src/java/oracle/weblogic/kubernetes/applications/clusterview/ClusterViewServlet.java

```diff
@@ -0,0 +1,150 @@
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NameNotFoundException;
+import javax.naming.NamingException;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import weblogic.management.jmx.MBeanServerInvocationHandler;
+import weblogic.management.mbeanservers.runtime.RuntimeServiceMBean;
+import weblogic.management.runtime.ClusterRuntimeMBean;
+import weblogic.management.runtime.ServerRuntimeMBean;
+
+/**
+ * Servlet to print all MBeans names and attributes in the server runtime.
+ */
+public class ClusterViewServlet extends HttpServlet {
+
+  Context ctx = null;
+  MBeanServer localMBeanServer;
+  ServerRuntimeMBean serverRuntime;
+
+  @Override
+  public void init(ServletConfig config) throws ServletException {
+    try {
+      ctx = new InitialContext();
+      localMBeanServer = (MBeanServer) ctx.lookup("java:comp/env/jmx/runtime");
+      // get ServerRuntimeMBean
+      ObjectName runtimeserviceObjectName = new ObjectName(RuntimeServiceMBean.OBJECT_NAME);
+      RuntimeServiceMBean runtimeService = (RuntimeServiceMBean) MBeanServerInvocationHandler
+          .newProxyInstance(localMBeanServer, runtimeserviceObjectName);
+      serverRuntime = runtimeService.getServerRuntime();
+      try {
+        ctx.lookup(serverRuntime.getName());
+      } catch (NameNotFoundException nnfe) {
+        ctx.bind(serverRuntime.getName(), serverRuntime.getName());
+      }
+    } catch (NamingException | MalformedObjectNameException ex) {
+      Logger.getLogger(ClusterViewServlet.class.getName()).log(Level.SEVERE, null, ex);
+    }
+  }
+
+  @Override
+  public void destroy() {
+    try {
+      ctx.unbind(serverRuntime.getName());
+      ctx.close();
+    } catch (NamingException ex) {
+      Logger.getLogger(ClusterViewServlet.class.getName()).log(Level.SEVERE, null, ex);
+    }
+  }
+
+  /**
+   * Processes requests for both HTTP <code>GET</code> and <code>POST</code> methods.
+   *
+   * @param request servlet request
+   * @param response servlet response
+   * @throws ServletException if a servlet-specific error occurs
+   * @throws IOException if an I/O error occurs
+   */
+  protected void processRequest(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    response.setContentType("text/html;charset=UTF-8");
+    try (PrintWriter out = response.getWriter()) {
+      out.println("<!DOCTYPE html>");
+      out.println("<html>");
+      out.println("<head>");
+      out.println("<title>ClusterViewServlet</title>");
+      out.println("</head>");
+      out.println("<body>");
```

Comment:
I'm not sure about this, but if you were to view this output in a browser, I suspect the newlines would be ignored and it'd all end up munged into one line. If so, you can put a <pre> after <body> and a </pre> before </body> to correct this.

Reply:
Added pre tag